docs/components.md

```diff
@@ -49,6 +49,7 @@ Independent, small visual UI elements to use in applications.
 
 ### Services
 
+* [service-contile](https://github.com/mozilla-mobile/android-components/tree/main/components/service/contile) - A library for communicating with the Contile services API.
```

Comment:
nit: we can add this to the [main README](https://github.com/mozilla-mobile/android-components/blob/main/README.md) too under services. 🙂